### DISCUSSION TITLE
Error "The resource was not found" in the Figma console

### DISCUSSION THREAD

**Discussion by PenseeNumerique** [unoplatform/uno#13424] [Q&A]
Hello, I wanted to generate a test page in Figma to learn how to use your tool. So, I followed the instructions for Figma in your documentation. I selected a template from your collection. Then, I wanted to view the XAML code on the 'Export' page, but there was no code displayed even though I entered the correct information in 'Properties'. I checked the 'Console' and found the error message 'The resource was not found: StatusBarForegroundColor'. I want to clarify that I haven't made any modifications, I simply retrieved the model and a template. Do you have any idea what could be causing this error?

Thank you in advance.
![TabTip_3QQpTh4HXW](https://github.com/unoplatform/uno/assets/103261801/2b898b9d-961c-4473-b77b-43139c4d7b26)
![TabTip_5o3jnXAQ04](https://github.com/unoplatform/uno/assets/103261801/76ee1159-a9e3-41fb-8ff8-cea352bddfd4)
![TabTip_RYV7AscKL5](https://github.com/unoplatform/uno/assets/103261801/8d316a1f-b0e7-4594-a2e8-3263a5b5064e)
![TabTip_z4PcDxTyZ1](https://github.com/unoplatform/uno/assets/103261801/82a38684-52c1-483e-a301-6df09f8ccbee)

**Comment by sasakrsmanovic** [comment]
Hello @PenseeNumerique . Just so we speed up the investigation, which tutorial did you follow? 

Thanks

Sasha

**Comment by PenseeNumerique** [comment]
Thank you for your response. I followed the tutorial from the Figma model => https://www.youtube.com/watch?v=5G73e21yN_w  and the one on your website https://platform.uno/docs/articles/external/figma-docs/learn/developers/plugin.html.

Edit : Video
![TabTip_9nC4iIfMgL](https://github.com/unoplatform/uno/assets/103261801/33932015-b508-48b2-a1f6-f99e7a735099)

And I noticed that if I click on "Export," I get the code in my clipboard, but there's no message indicating that it has been copied there. Also, I don't see the formatted code. but Only for Colors, Fonts, and Localization File

**Reply by iurycarlos** [reply]
Hello @PenseeNumerique !
As I could see, the Code Editor is not visible somehow. 
I'm using Chrome as well and I couldn't reproduce the issue following your steps:

![image](https://github.com/unoplatform/uno/assets/6876973/c42a6be8-d64f-4cc7-bed9-815d5ea50b6e)
 
Could you please press CTRL + SHIFT + J to open console and send a print of it?

Did you tried at Figma desktop version? Or, do you mind to try using another browser?

**Reply by sasakrsmanovic** [reply]
@PenseeNumerique  - what browser did you use please?

**Comment by iurycarlos** [comment]
Hi @PenseeNumerique!
Did you have any progress?
